README.md

```diff
@@ -31,10 +27,22 @@ Example: `.eslintrc.js`:
 
 ```javascript
 module.exports = {
-  extends: ['@twostoryrobot/eslint-config/react']
+  extends: ['react-app', '@twostoryrobot/eslint-config/react']
 }
 ```
 
+Optionally, if you are using create-react-app, you can change the `eslintConfig`
```

Comment:
Hmmm good catch, projects that follow the above instructions end up with an `eslintConfig` object left in the package.json that is ignored.